samples/web/content/apprtc/gcmrecord.py

```diff
@@ -28,19 +28,27 @@ class GCMRecord(ndb.Model):
   last_modified_time = ndb.DateTimeProperty()
 
   @classmethod
-  def get_by_user_id(cls, user_id):
-    q = GCMRecord.query(ancestor=get_ancestor_key())
-    return q.filter(GCMRecord.user_id == user_id).fetch()
+  def get_by_user_id(cls, user_id, verified = False):
```

Comment:
This, and the other APIs that take 'verified', returns all records if verified == False, and returns verified records if verified == True, which may be not what one would expect from the interface. 
Maybe rename 'verified' to 'verified_only'.